bundle/tests/scorecard/kuttl/service-certificate/02-assert.yaml

```diff
@@ -1,8 +1,33 @@
+  tls.key: LS0tLS1CRUdJTiBSU0EgUFJJVkFURSBLRVktLS0tLQpNSUlFcEFJQkFBS0NBUUVBdGFYT3FTKzkwWVA0V0FuWnlLZHV0cEROZ1p1a1ZtN2FaNmZ6VURoT3h3VXc0T1NyCnp3OE9CcEY1d1dLNUlZTExHa2x1Rno0RGY3Tlk5RUdHdHdtd1BQYnAzeENvdnVMa3AwVFpMcUMzVEZMcDJmOG8KK3dFUTl2L0RIUnZadk8xSUcyNzhFSDhyODhxZlJEQnpqdEFZcjFhYWNWczZOVXlIblEvWmZhNXZ5bTRsYnNkVApGVTJHbGdxbm9qeVBTZzh6RFpjbzUzcUhlUVY1d0lvbmhLNTB6SHlVMUVUaFdpRWlNeUZSbXN5OWxXcDdBYlJvCklMUkVaMXVhdnVzenNNRTdoUEhXbzRkNGVkbURwRXFSTTk1OTRVUEhZeGUzVDJYVUpDbkRCd2dWQzY1RzJCOEYKMEo3WE5UY1RjZGhBdVA5a2NkQTY0bktpVGZWOC9BU1dLNE1tY1FJREFRQUJBb0lCQVFDcFBkd0h6RUI2SmhCTAp6cG5JRngvVEg2VXBuZDhHM08vRzVqZVB4WGE0ZllDbCtNaU8rZ0xpWWNaa1JOd2llZDlNRmFCcUxWQkxPY21tCmdpckcvVkxqNmxYS2pKWDhqQi9vTFFDQkxpSzZEUUp2SndxQ211SzRqMkcyREUzWmRlYmFFSlR2dGQwNG5zKzMKVThGSERXSUNjNWtnQnBSOE5scWdIclZRSVNkdEhaTVB6U25aaHBmUno4elNtNkcwTTdneUZBbjlZQWM4OWRDNApXVmlSYjBrRmlHTFhVNWlUNG81WEptNHh1Rjc2QWxKZ3k3b2dXbTNqNmd5RWhzallGNTdYeVYvb2RFcFdtZVFBCkZlTGdtNzU2TUtCcW53SE9kT011TlJDTk8yYmZET2pjWXJEVkUwanR2WG5JdjAxVnlSeDVlU09lWTM5QmdPNFkKbDcxYUdiZEJBb0dCQU5sT0Y5L1lsSktGclg4VkdRMlpCNjN4MmFVUU9sN1J3cytkb3p6NWMwVGhEY1dHajdNSQpIRHgzcHNnaWhxQnlXMGhsSkcwMDY4R3JpMzhmeXNrRlZ2NmJJUnZNTEFKL0VMMGV6NzE0eTJrbHlJZGZXZW1pCnBuc0x4d3puWjlTdGllK3loUzNuVCtMd21hbWdydU9ubC9ORURzVVhEYXg0dTIrMXRqYjI3bGg1QW9HQkFOWCsKUTRUbTBOSHJFNyt1a0ZiUzJET1BjZGtUejhhRysxNG5pZWF0NVVqWVVsUVBxU0hMbC9EbDlWU2RMYWZuWTlvRAp1ai9qUmpiWThWK3hlZDhCSUFhNUk2Rkh0VFBySEpUajdmZDJMQUpvTnBQOWErZWUwMG9LRytlaHdoRmtpM3hDCkJpUERlWHV3MStTS01LbVNxR21pU05MczFXQkV4YnhyWmV2cmZ5KzVBb0dBZDg3em9Nb0RXQ2wxdDV4RlBOY0QKZ2ViVzdZZmQ5c3RGMGp3Q2RTcS9YOW1KUXo5bVdQbit2MlBrdzNFdHNxWjZKNWNFRjJYWEdlZGZ1TTRwZXQyRAowa3ByWGEwYkVxRkY5RTlaa0FoYWRmV0w4djhpVG9zTHFNbWNqRVBGWGwvenJnRktKajNqTUtjZ3dZSkRDbWNyCmdzY2lNWTM5VHZzZHRJQ0dIeFhUNFZrQ2dZRUFxTjc4WFhiZ3Q2ZXhjQVRMOXhod2NVL3o0Z0RXYUs3QTAyNlUKVHF4UVVvb1RmaTdWbzh2TTNkZzdJRWpoYkswWGlVYUR4a01PZDkzL3JTcWJLRGIxRlJrcFd1OEJ1alVCMnlWNwo5MXNVSTlvbHArZkhMWU9TN2N6SWRLU2NSNmwyd2RwQWloeUtVK2djbDBmT1d5TlhML1ZYVHN5REhuTm5MK2Y1CkNKSlJUbEVDZ1lCL20rNnB3bzEvaTV5UXhXRlhNemZibEtkKzlEUXVFbCt4Si9TZUYrZUNpUkhqN0NULzk3Rm8KSjNqTGdzQmFWRjhlNWRhRU9uMkVwTEhkRzJUYnRoVy8vQVRVc1hVbzRncXBENWM4TXZqUWcwRnloVlFGdzIzKwpTQXhnK3VMZitvazZLVVRnSGQxZEU1NnRoYWFJdUNsRHg0Z2xJdkswNlVodDZHVXF6YWVQRnc9PQotLS0tLUVORCBSU0EgUFJJVkFURSBLRVktLS0tLQo=
+---
 apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: service-with-cert-lib-app
 status:
+  observedGeneration: 2
   replicas: 1
   readyReplicas: 1
   updatedReplicas: 1
+spec:
+  template:
+    spec:
+      volumes:
+        - name: svc-certificate
+          secret:
+            defaultMode: 420
+            secretName: service-tls-secret
```

Comment:
In addition to volume, we should check that volumeMount _svc-certificate_ is also specified in Deployment. But that's a separate issue. RCO would also need that update. I'll open an issue.